src/cppgc.rs

```diff
@@ -479,16 +493,22 @@ macro_rules! member {
           self.inner.assign(ptr);
         }
 
-        #[doc = "Borrow the object pointed to by this "]
+        #[doc = "Get the object pointed to by this "]
         #[doc = stringify!($name)]
-        #[doc = "."]
-        pub fn borrow(&self) -> Option<&T> {
+        #[doc = ", returning `None` if the pointer is empty or has been garbage-collected."]
+        #[doc = ""]
```

Comment:
are these newlines? i think they're just concatted. maybe just use a multiline string.

Reply:
Yes, they are newlines. Doc attributes are interpreted the same way as lines in doc comments, so this is the usual approach to formatting them in macros, but I'm fine with whatever you think is best.